readthedocs/organizations/views/public.py

```diff
@@ -69,6 +67,10 @@ def get_context_data(self, **kwargs):
             context["owners"] = org.owners.all()
 
         context["projects"] = projects
+        context["notifications"] = Notification.objects.for_user(
+            self.request.user,
+            resource=org,
+        )
```

Comment:
Does something like `org.notifications.for_user(...)` not work?

Reply:
I think it works, but we still need to pass `resource=org` anyways, right? Otherwise, we would need to add more complexity in the underlying `.for_user()` queryset method I guess.

Reply:
when using the manager that way, you don't need to pass `resource`, since the queryset will be filtered already.